src/averageAges.js

```diff
@@ -36,8 +40,18 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const newArray = people.filter(person =>
+    withChildren
+      ? person.sex === 'f' && people.find(child => child.mother === person.name)
+      : person.sex === 'f'
+  );
+
+  const womenAverageAge = newArray.reduce((sum, woman) =>
+    sum + woman.died - woman.born, 0) / newArray.length;
+
+  return Math.round(womenAverageAge * 100) / 100;
```

Comment:
i think we need to make this calculation 
```suggestion
  return womenAverageAge / newArray.length;
```